### PR TITLE
Go-To-Dash update

### PR DESCRIPTION
**Pull request by AprilSylph**
Update the View button for the 1st August iframe update

image

### CODE REVIEW

**Comment by nightpool** [comment]
"View" should probably say something like "On Dash".

can we add all of the css classes so it doesn't break the existing iframe layout?

does this get collapsed by default or not?

**Comment by AprilSylph** [comment]
Done

Unfortunately no; adding back the class that makes it fit with old iframe layout would put the embed icon back on it in the new iframe

Yes

image
image

How's this?

File: Extensions/xkit_patches.js [hunk 1]
```diff
@@ -538,39 +538,47 @@ XKit.extensions.xkit_patches = new Object({
...
-				return $(".btn.reblog,.tx-button.reblog-button");
+				return this.tx_button_selector("reblog");
 			},
 
 			/**
 			 * @return {JQuery} The dashboard button in the iframe
 			 */
 			dashboard_button: function() {
-				return $(".btn.dashboard,.tx-button.dashboard-button");
+				return this.tx_button_selector("unfollow");
```
Comment (nightpool) [inline Extensions/xkit_patches.js]: i copy and pasted this wrong, it should be `dashboard`

Comment (AprilSylph) [inline Extensions/xkit_patches.js]: whoops

**Comment by AprilSylph** [comment]
Integrated/Resolved

**Comment by nightpool** [comment]
Integrated into #1380, which just got merged. Thanks @AprilSylph!

**Pull request closed without merging**